sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-decisions.dmn

```diff
@@ -1,3121 +1,3121 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Correct me if wrong is the full time decision part of the ticket?

Reply:
@lewischen-aot do you need some help reverting this? It would be good to just have it reverted if no changes are needed here.